script.js

```diff
@@ -0,0 +1,95 @@
+let spinnerLoad = document.getElementById("sppinner");
+let inputField = document.querySelector("#sequenceInput");
+const errorDivPlaceholder = document.querySelector("#error-div-placeholder");
+const calcDiv = document.querySelector("#calc-div");
+let btn = document.querySelector("#resultBtn");
+const errorDivOver50 = document.createElement("div");
+errorDivOver50.innerHTML = "Can't be larger than 50";
+errorDivOver50.setAttribute("id", "error50");
+errorDivOver50.classList.add(
+  "bg-danger",
+  "border-danger",
+  "text-danger",
+  "bg-opacity-25",
+  "rounded-1",
+  "error-box50"
+);
+
+function fibcalc(sequence) {
+  let url = "http://localhost:5050/fibonacci/" + sequence;
+  fetch(url)
+    .then((response) => {
+      if (response.ok) {
+        return response.json();
+      }
+      throw new Error("Server Error: 42 is the meaning of life");
+    })
+    .then((data) => {
+      calcList();
+      document.getElementById("resultNum").innerHTML = data["result"];
+    })
+    .catch((error) => {
+      cleanUp();
+      let errorDiv42 = document.createElement("div");
+      errorDiv42.innerHTML = "Server Error: 42 is the meaning of life";
```

Comment:
You already made this in line 25, you can use the `error` paramater and it will be the same. Although again, you should not create the error, it should come from the server: `errorDiv42.innerHTML = error;`